### ISSUE TITLE
 Add pillar in room

### ISSUE THREAD

**Issue by Novanef** [ditrit/OGrEE-CLI#121]
add pillar as room attribute like separator

**Comment by hchybz** [comment]
```
 "pilars" : [ 
									{"centerXY" : [41.1, 7.3],  "sizeXY": [0.75,0.75], "rotation" : 0  },
```

**Comment by hchybz** [comment]
`DEMO.SITE.BD.ROOM:pilar=[41.1,7.3]@[0.75,0.75]@12`

**Comment by denizul** [comment]
Added as of 60947c84ec on issue121Pillar

**Comment by Cedrok** [comment]
When we create a room with a template containing pillars, the CLI doesn't add those pillars in the "attributes"

**Comment by denizul** [comment]
Template parsing was merged into master as of 79a56620c
